pkg/cmd/target/set.go

```diff
@@ -169,3 +192,114 @@ var TargetSetCmd = &cobra.Command{
 		return nil
 	},
 }
+
+func handleTargetJSON(data []byte) error {
```

Comment:
![Screenshot 2024-12-10 at 17 06 16](https://github.com/user-attachments/assets/ba45f3cb-3b0c-4fd1-b5d0-efc1447d2d17)

The disabled predicate doesn't work. In the case from the screenshot, it should throw an error because i defined `Remote Hostname` which is disabled since the target is named `local`

Reply:
@Philip-21 this is still not addressed

Reply:
i created a ` DisabledPredicateOptions` for hadling the required options for local target without hardcoding